src/SignalR/clients/csharp/Http.Connections.Client/src/HttpConnection.Log.cs

```diff
@@ -66,6 +66,9 @@ private static class Log
             private static readonly Action<ILogger, HttpTransportType, Exception> _transportStarted =
                 LoggerMessage.Define<HttpTransportType>(LogLevel.Debug, new EventId(18, "TransportStarted"), "Transport '{Transport}' started.");
 
+            private static readonly Action<ILogger, Exception> _serverSentEventsNotSupportedByBrowser =
```

Comment:
I think you're missing the method that uses this

Reply:
Yeah just noticed that, added it in the newest commit